compiler/rustc_session/src/options.rs

```diff
@@ -1978,6 +1978,9 @@ options! {
         "enable queries of the dependency graph for regression testing (default: no)"),
     randomize_layout: bool = (false, parse_bool, [TRACKED],
         "randomize the layout of types (default: no)"),
+    regparm: Option<u32> = (None, parse_opt_number, [TRACKED],
+        "On x86-32 targets, the regparm N causes the compiler to pass N arguments \
+        in registers EAX, EDX, and ECX instead of on the stack."),
```

Comment:
```suggestion
        in registers EAX, EDX, and ECX instead of on the stack.\
        It is UNSOUND to link together crates that use different values for this flag!"),
```